workflows/raredisease.nf

```diff
@@ -150,6 +157,14 @@ workflow RAREDISEASE {
         PREPARE_GENOME.out.fasta
     )
 
+    // STEP 1.6: EXPANSIONHUNTER
+    CALL_REPEAT_EXPANSIONS (
+            ch_marked_bam.join(ch_marked_bai, by: [0]),
+            PREPARE_GENOME.out.fasta,
+            params.genomes.GRCh38.variant_catalog
```

Comment:
@ljmesi This works but since there'a utility function to decompose a nested dictionary (https://github.com/nf-core/raredisease/blob/dev/lib/WorkflowMain.groovy#L85), we can just use this instead
```suggestion
            params.variant_catalog
```
We'll need to add an extra line in https://github.com/nf-core/raredisease/blob/dev/main.nf at line 23
`params.variant_catalog   = WorkflowMain.getGenomeAttribute(params, 'variant_catalog')`